org.eclipse.dawnsci.nexus/src/org/eclipse/dawnsci/nexus/NexusFile.java

```diff
@@ -28,7 +28,10 @@
  */
 public interface NexusFile extends AutoCloseable {
 
-	public static final String NXCLASS = "NX_class";
+	/**
+	 * @deprecated Use {@link NexusConstants#CLASS}
+	 */
+	public static final String NXCLASS = NexusConstants.CLASS;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @Deprecated annotation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)